src/sidebar/services/test/session-test.js

```diff
@@ -148,6 +150,22 @@ describe('SessionService', () => {
         });
       });
 
+      it('should reject if the profile fetch repeatedly fails', async () => {
+        const fetchError = new Error('Server error');
+        fakeApi.profile.read.rejects(fetchError);
+
+        const session = createService();
+        let error;
+        try {
```

Comment:
I think we can just make this a 1-liner

`await assert.rejects(session.load(), fetchError.message);`

Reply:
Done! - I did forget to use `await` when I first made this change to I had to push a follow-up commit. This issue (an atypical usage pattern) makes me thing that we probably shouldn't be monkey-patching this method onto `assert` and instead just import the helper explicit from some module.